src/components/LessonPage/MainLanguageButton.js

```diff
@@ -37,12 +44,12 @@ const mapStateToProps = (state, {course, lesson, isReadme}) => {
   const tf = getTranslateFilter(state);
   const language = state.language; // E.g. 'en'
   const lang = tf('language', language); // Name of language, e.g. 'English'
-  const {path} = getLessonFrontmatter(course, lesson, language, isReadme);
-
+  const {path, external} = getLessonFrontmatter(course, lesson, language, isReadme);
   return {
```

Comment:
Forslag: Definer en `const enabled = path && !external` her, og send den inn som prop istedet for `external` (og bruk den i buttonText under her). Så definerer du `disabled: !enabled` i options.